### PR TITLE
Unify tracing story for TestHost

### PR DESCRIPTION
**Pull request by rynowak**
- Adds a TraceSource
- Use the TraceSource for everything
- Detect environment variable to ramp up verboscity
- Clean up dead/duplicated code

### CODE REVIEW

File: src/Microsoft.Dnx.TestHost/Program.cs [hunk 1]
```diff
@@ -114,16 +115,18 @@ public int Main(string[] args)
 
                             var testServices = TestServices.CreateTestServices(_services, project, channel);
                             await ProjectCommand.Execute(testServices, project, "test", commandArgs);
-
-                            Trace.TraceInformation("[ReportingChannel]: OnTransmit(DiscoverTests)");
```
Comment (rynowak) [inline src/Microsoft.Dnx.TestHost/Program.cs]: These are all redundant. The Channel does this also.

File: src/Microsoft.Dnx.TestHost/Program.cs [hunk 2]
```diff
@@ -141,191 +144,33 @@ public int Main(string[] args)
...
-                        Thread.Sleep(10 * 1000);
-                    }
-
-                    var projectPath = projectOption.Value() ?? env.ApplicationBasePath;
-                    var port = int.Parse(portOption.Value());
-
-                    return await ExecuteTests(port, projectPath, tests.Values);
-                });
-
-            });
```
Comment (rynowak) [inline src/Microsoft.Dnx.TestHost/Program.cs]: This is dead code that we had planned to remove when VS went RTM.

**Comment by Eilon** [comment]
Seems straightforward, :shipit:

**Comment by davidfowl** [comment]
Why not use ILogger?

**Comment by rynowak** [comment]
> Why not use ILogger?

That would bring in more dependencies than necessary.We'd have to pull in the console logger.

**Pull request closed without merging**